cmd/dagger/cmd/project/init.go

```diff
@@ -41,6 +41,7 @@ var initCmd = &cobra.Command{
 		if err != nil {
 			lg.Fatal().Err(err).Msg("failed to initialize project")
 		}
+		fmt.Println("project initialized! To install dagger universe, run `dagger project update`")
```

Comment:
Start with "**P**roject [...]". Not sure about "dagger universe" because it also install _core_. Maybe "dagger packages"?

Reply:
Solved in https://github.com/dagger/dagger/pull/2119/commits/bc875473906fad73f57a73c8fe31a269067d734e